R/pipe_continuation_linter.R

```diff
@@ -3,14 +3,56 @@
 #' Check that each step in a pipeline is on a new line, or the entire pipe fits on one line.
 #'
 #' @evalRd rd_tags("pipe_continuation_linter")
+#'
+#' @examples
+#' # will produce lints
+#' code_lines <- "1:3 %>%\n mean() %>% as.character()"
+#' cat(code_lines)
```

Comment:
oh, actually this should be `writeLines()`, come to think of it, since `code_lines` doesn't have a trailing `\n`... let's fix more broadly in a follow-up?

Reply:
The change was easy enough, so included in this PR itself.

Reply:
i meant that we also used `cat()` elsewhere in the sister PRs, e.g.

https://github.com/r-lib/lintr/blob/ab8b77d006b10d56af5054aea288f9f9d35f1bf7/R/brace_linter.R#L22

so we should do a sweep to use `writeLines()` more broadly

Reply:
Yes, done this more broadly in this PR itself.